java-frontend/src/test/java/org/sonar/java/resolve/SymbolTableTest.java

```diff
@@ -1222,8 +1223,7 @@ public void infer_stream_types_on_chained_map() {
 
     IdentifierTree map = result.referenceTree(63, 8);
     MethodJavaType methodJavaType = (MethodJavaType) map.symbolType();
-    // expression type is correctly infered but method type is not recomputed and thus is still defered
-    assertThat(methodJavaType.resultType.isTagged(JavaType.DEFERRED)).isTrue();
+    assertThat(methodJavaType.resultType.isTagged(JavaType.DEFERRED)).isFalse();
```

Comment:
This is actually _really_ important  : this is the kind of uncomplete udpate that can lead to unfailing tests after some changes. This test is now lacking assertions.